user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -133,12 +133,30 @@ settings.
 ### Getting to the boot screen
 
 Just after you power on your machine, make the Qubes OS medium available to the
-computer by inserting your DVD or USB drive. Shortly after the Power-on
-self-test (POST) is completed, you should be greeted with the Qubes OS boot
-screen. 
+computer by inserting your USB drive.
+
+Then you'll need to power on the computer and immediately press a specific key to
+enter what's called the "BIOS" of you computer. The key to press varies wildly from
+one computer to another. `Esc`, `Del` and `F10` are
+common keys. If you're not sure about the key to press you'll have to view in the
+machine's user guide or serch the web for `accessing BIOS <COMPUTER_MODEL>` 
+(replacing with your specific computer model).
+
+The BIOS menu will looks something like the following pictures:
+
+| ![](/attachment/doc/Thinkpad-t430-bios-main.jpg)  | |
+|----|----|
+| [CC-BY-SA-4.0 Vitaly Zdanevich](https://commons.wikimedia.org/wiki/File:Thinkpad-t430-bios-main.jpg) | ADD here UEFI
+
+Within the BIOS you'll navigate to the boot menu and either select to boot from your USB device or order the boot devices in such a way that the USB stick is the first one. Then save the configurations and restart your computer. After this the computer should show the Qubes boot screen:
 
 ![Boot screen](/attachment/doc/boot-screen.png)
 
+<div class="alert alert-info" role="alert">
+  <i class="fa fa-info-circle"></i>
+  <b>Note:</b> Please note that there may be other thing you need to do in BIOS. *You may need to change them later* if the installer still refuses to start (deactivating secure boot can solve it) or if when installing Qubes shows an error saying you computer does not support virtualizaiton technology (where you'll need to activate virtualization technology).
+</div>
```

Comment:
I think it might look somewhat strange to have two info alerts back-to-back like this. What do you think?

Reply:
I was also thinking that. But since github doesn't allow me to draft up a thing before opening a PR it went as it was. I'll deal with that in a future commit.

Reply:
removed this entirely.